### PR TITLE
DEV-25706: add optional sort query parameter to get all users section

### PR DESCRIPTION
**Pull request by george-haddad**
- Add optional `sort` query param to get all users section

### CODE REVIEW

**Review by tkrisztian95 — COMMENTED** [review]
Looks good! Added few comments to improve the rendered doc visual layout.

File: apiary.apib [hunk 1]
```diff
@@ -1285,10 +1285,38 @@ Returns information about the current user. See below for possible responses.
 
 ## User Resource [/api/v1/user]
 
-### Get All Users [GET /api/v1/user]
+### Get All Users [GET /api/v1/user{?sort}]
+
+This request will retrieve a list of all users in their natural order, this means in the order they were inserted.
+The list may only be sorted by 1 field and if the `sort` query parameter is used. The query parameter can be
```
[the remark this answers is not in the document]
Comment (george-haddad) [reply apiary.apib:1291]: I like the split into sections, thanks.

File: apiary.apib [hunk 2]
```diff
@@ -1285,10 +1285,38 @@ Returns information about the current user. See below for possible responses.
...
+
+- May only sort by 1 field
+- Query parameter max length is 50 characters, excluding order direction prefixes '+' and '-'
+- Query parameter cannot be empty or filled with blank spaces
+- Query parameter can only accept mixed case alpha numeric characters and underscores
+- Omitting the prefix will automatically assume ascending order
 
 + Request Get a list of all users (application/json)
 
++ Parameters
```
[the remark this answers is not in the document]
Comment (george-haddad) [reply apiary.apib:1304]: wow nice. I spent quite a bit of time just getting the right syntax! so this is a welcomed improvement.

**Review by tkrisztian95 — APPROVED** [review]
Looks good! Nice work!

**Pull request merged**